Library/Homebrew/cask/blacklist.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Cask
+  module Blacklist
+    WHITELISTED_CASKS = %w[
```

Comment:
I believe these are all the existing `adobe` casks that are still valid.

Reply:
The blacklist of adobe casks is shorter than this whitelist, and this will need updating if newer adobe casks are introduced.

Might be better to change the conditional further down to `/^adobe-(after|illustrator|indesign|photoshop|premiere)/

Reply:
Ah ok! I didn't know there was a defined blacklist for adobe casks. I was under the impression it could be any (besides the whitelisted ones).